driver/node.go

```diff
@@ -60,11 +63,34 @@ func (d *QuobyteDriver) NodePublishVolume(ctx context.Context, req *csi.NodePubl
 		}
 	}
 	var mountPath string
-	if len(volParts) == 3 { // tenant|volume|subDir
-		mountPath = fmt.Sprintf("%s/%s/%s", d.clientMountPoint, volUUID, volParts[2])
+	if d.IsQuobyteAccesskeysEnabled {
+		podUID := getSanitizedPodUIDFromPath(targetPath)
+		accesskeyID, ok := secrets["access_key_id"]
+		if !ok {
+			return nil, fmt.Errorf("Mount secrets should have 'access_key_id: <access_key_id>'")
```

Comment:
Mount secrets or secret? Here and below.

Reply:
Changed it to secret for logging.
k8s mostly uses plural. See line above "secrets["access_key_id"]"